### ISSUE TITLE
Find in Folder uses the monorepo package name instead of actual path

### ISSUE THREAD

**Issue by kaaloo** [folke/vscode-monorepo-workspace#61]
When right clicking on a package name in my monorepo and selecting "Find in Folder", the "files to include" field is populated with a path that uses the extensions package name (complete with emoji) instead of the actual path. This is in vscode 1.50 (first noticed it in 1.49) and monorepo-workspace extension 1.2.0.

**Comment by folke** [comment]
This seems to be an upstream issue in vscode. The same issue arrises when manually creating / editing a workspace and configuring the name. Other functionality that uses workspace folders seems to be working.

Probably best to submit this issue upstream.

**Comment by kaaloo** [comment]
Ok I found the upstream issue: https://github.com/microsoft/vscode/issues/65402  All works well except if you have a slash in the workspace folder name. :man_shrugging:

**Comment by folke** [comment]
That seems to be an issue from 2018, so it must have been fixed at some point.
I also just checked and even with folders that don't have a `/`, the issue happens.

Checking the vscode, the problem is the line at:

https://github.com/microsoft/vscode/blob/c7c471bdde61781c22ff16497c1d0af5e625f441/src/vs/workbench/contrib/search/browser/search.contribution.ts#L420

```js
folders.push(result.stat.isDirectory ? result.stat.resource : dirname(result.stat.resource));
```

it should be something like the following instead:
```js
folders.push(result.stat.isDirectory ? result.stat.resource.fsPath : dirname(result.stat.resource.fsPath));
```

I'll create a new issue in vscode for this, to get it fixed.
